decrypt_connector_auth.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# Decrypt the file
+mkdir $HOME/target/
+
+mkdir $HOME/target/test
```

Comment:
```suggestion
mkdir -p $HOME/target/test
```

This would create the required parent directories and not error out if `$HOME/target/test` already exists.